pkg/core/blockchain.go

```diff
@@ -1240,10 +1241,38 @@ func (bc *Blockchain) verifyTx(t *transaction.Transaction, block *block.Block) e
 			return ErrMemPoolConflict
 		}
 	}
+	if err := bc.verifyTxAttributes(t); err != nil {
+		return err
+	}
 
 	return bc.verifyTxWitnesses(t, block)
 }
 
+func (bc *Blockchain) verifyTxAttributes(tx *transaction.Transaction) error {
+	for i := range tx.Attributes {
+		switch tx.Attributes[i].Type {
+		case transaction.HighPriority:
+			pubs, err := bc.contracts.NEO.GetCommitteeMembers(bc, bc.dao)
+			if err != nil {
+				return fmt.Errorf("%w: %v", ErrInvalidAttribute, err)
```

Comment:
Well, it's not a tx fault, so I don't think `ErrInvalidAttribute` is appropriate here.